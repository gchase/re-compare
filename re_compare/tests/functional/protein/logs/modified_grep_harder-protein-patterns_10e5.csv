regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,1231,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,1231,"[-1, -1]"
DHKNLD.D,first_match,0,998,"[-1, -1]"
DHKNLD.D,all_matches,0,998,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,2137,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,2137,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,10,"[99267, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,2147,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,0,1418,"[-1, -1]"
(GI)(G|R)P.Y..K..R,all_matches,0,1418,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,0,1584,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,0,1584,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,0,1917,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,0,1917,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,0,2213,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,0,2213,"[-1, -1]"
P..L...KWE.C,first_match,0,1197,"[-1, -1]"
P..L...KWE.C,all_matches,0,1197,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,1624,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,1624,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,0,1315,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,0,1315,"[-1, -1]"
