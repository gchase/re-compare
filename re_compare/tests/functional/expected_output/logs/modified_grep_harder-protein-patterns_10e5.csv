regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,1545,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,1545,"[-1, -1]"
DHKNLD.D,first_match,0,1445,"[-1, -1]"
DHKNLD.D,all_matches,0,1445,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,1942,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,1942,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,12,"[99267, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,1954,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,0,1536,"[-1, -1]"
(GI)(G|R)P.Y..K..R,all_matches,0,1536,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,0,3018,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,0,3018,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,0,1105,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,0,1105,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,0,1922,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,0,1922,"[-1, -1]"
P..L...KWE.C,first_match,0,1285,"[-1, -1]"
P..L...KWE.C,all_matches,0,1285,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,1182,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,1182,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,0,1664,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,0,1664,"[-1, -1]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),first_match,1,2174,"[41350, 41358]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),consecutive,1,2174,"[41350, 41358]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),all_matches,1,2977,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),first_match,0,1171,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),all_matches,0,1171,"[-1, -1]"
(A|G....GK)(S|T),first_match,1,960,"[56, 57]"
(A|G....GK)(S|T),consecutive,1,960,"[56, 57]"
(A|G....GK)(S|T),consecutive,2,7,"[93, 94]"
(A|G....GK)(S|T),all_matches,2,967,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,first_match,0,1429,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,all_matches,0,1429,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,first_match,0,1272,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,all_matches,0,1272,"[-1, -1]"
